COPYRIGHT

```diff
@@ -0,0 +1,7 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```

Comment:
I'm "meh" on this...it doesn't seem to add much information that isn't already in the git repo.  But, not a problem either.

Reply:
I can leave this one out, but it replaces those ubiquitous boilerplate copyright lines (that I dropped here).